website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,298 @@
+provided by the Controllers. Only clients provided by cluster-scoped projects are able to manage cluster-scoped CRD's. 
+For further information see: [CRD scope doc][crd-scope-doc].
+
+## Watching resources in all Namespaces (default)
+
+A [Manager][ctrl-manager] is initialized with no Namespace option specified, or `Namespace: ""` will watch all Namespaces. In the `main.go` you will 
+be able to find it. See:
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+})
+...
+```
+
+**NOTE** The above example is the default code generated with `operator-sdk init` which initializes the Manager to be cluster-scoped.
+
+## Watching resources in a single Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to a specific namespace set the Namespace attribute from [Options][ctrl-options]:
+
+```go
+...
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    Namespace:          "operator-namespace",
+    MetricsBindAddress: metricsAddr,
+})
+...
+``` 
+
+## Watching resources in a set of Namespaces
+
+It is possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] attribute from 
+[Options][ctrl-options] to watch and manage resources only in a specific set of namespaces:
+
+```go
+...
+namespaces := []string{"foo", "bar"} // List of Namespaces
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a namespace that 
+is not in this List of `namespaces` then, its controller will not re-trigger its reconcile since the namespaced is NOT 
+part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate. 
+Also note that you may face performance issues when using this with a high number of namespaces.
+
+## Restricting Roles and permissions
+
+The operator-scope defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. After updating the Manager's scope to be Namespaced, the [Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] permissions should be restricted accordingly.
+
+In the projects, the permissions are specified in the directory `config/rbac/`. The RBAC manifest at `config/rbac/role.yaml` and `config/rbac/role_binding.yaml` specifies the permissions needed by the operator to manage its resources, and is a `ClusterRole` by default. 
```

Comment:
The first sentence is redundant. We're already highlighting that the RBAC manifests are in `config/rbac/...` in the next one.

Also the rolebinding manifest does not specify permissions. But if you want to mention `rolebinding.yaml` here then you need to rephrase the sentence a bit.
```suggestion
The `ClusterRole` manifest at `config/rbac/role.yaml` specifies the permissions needed by the operator to manage its resources. 
The `ClusterRoleBinding` manifest at `config/rbac/role_binding.yaml` binds that to the operator's `ServiceAccount`.
```

Reply:
Addressed your comment by trying to add an smaller info yet.